### PR TITLE
Md player initialize

### PR DESCRIPTION
**Pull request by margdaly**
- Created #initialize for Player class 
    - @symbol
    - @input 

- Let me know if I need to add more expects

### CODE REVIEW

File: spec/player_spec.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+require 'spec_helper'
+
+RSpec.describe Player do
+  describe '#initialize' do
+    it 'exists' do
+      player = Player.new
+
+      expect(player).to be_a(Player)
+      expect(player.symbol).to be_a(String)
```
Comment (grace-joh) [inline spec/player_spec.rb]: What do you think about expecting `player.symbol` to eq `'X'`? Would we need to specify if the symbol is a string of the letter/character we want? Curious about your thoughts!

File: spec/player_spec.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+      expect(player.input).to eq(nil)
+    end
+  end
```
Comment (grace-joh) [inline spec/player_spec.rb:20]: Also, what are your thoughts on a second/edge case(?) test for #initialize so that "first test is not a coincidence"?? I've seen the instructors demo it in class a few times so I also started adding it, but do we want to consider it for our tests? 

Otherwise, these are the three expects I would have done, so looks good to me! :)

File: spec/player_spec.rb (hunk 1 above)
Comment (margdaly) [reply spec/player_spec.rb]: Noted! I think checking for "X" is a better test.

Comment (grace-joh) [reply spec/player_spec.rb]: Sounds good! Will resolve!

File: spec/player_spec.rb (hunk 2 above)
Comment (margdaly) [reply spec/player_spec.rb:20]: I would like to test for the second edge case!

Comment (grace-joh) [reply spec/player_spec.rb:20]: Okay! I will add the second test for Computer class as well!

**Comment by margdaly** [comment]
Fixed the initialize test to eq 'X' instead of string 
Added edge case for initializing second object

**Review by grace-joh — APPROVED** [review]
LOOKIN GOOOD
